Dockerfile

```diff
@@ -60,20 +60,26 @@ RUN apt-get install -y \
 
 # get the latest nightly ROS2 build -> ros2_ws/ros2_linux
 WORKDIR /ros2_ws
-RUN wget -nv https://ci.ros2.org/view/packaging/job/packaging_linux/lastSuccessfulBuild/artifact/ws/ros2-package-linux-x86_64.tar.bz2
+RUN wget -nv -t 5 https://ci.ros2.org/view/packaging/job/packaging_linux/lastSuccessfulBuild/artifact/ws/ros2-package-linux-x86_64.tar.bz2
 RUN tar -xjf ros2-package-linux-x86_64.tar.bz2
 
 # clone navigation2 repo
 WORKDIR /ros2_ws/navigation2_ws/src
-RUN git clone https://github.com/ros-planning/navigation2.git
+ARG USER=ros-planning
+RUN git clone https://github.com/$USER/navigation2.git
 
 # change to correct branch if $BRANCH is not = master
 WORKDIR /ros2_ws/navigation2_ws/src/navigation2
 ARG PULLREQ=false
+ARG BRANCH=master
 RUN echo "pullreq is $PULLREQ"
 RUN if [ "$PULLREQ" == "false" ]; \
```

Comment:
I'd prefer if you made `if [ $PULLREQ == "false" && $BRANCH == "master"]; then `  (echo the message) be the base case, but add a new case for `elif [ $BRANCH != "master" ]; then` checkout the branch. 

I realize the end effect is the same but it makes the default case do nothing instead of checking out master onto a temporary branch which is more clear, I think it also is more readable and understandable in the $BRANCH case.